cmd/run.go

```diff
@@ -268,6 +268,11 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) error {
 	logger.Debug("Waiting for engine processes to finish...")
 	engineWait()
 	logger.Debug("Everything has finished, exiting k6!")
+	if test.keywriter != nil {
+		if err := test.keywriter.Close(); err != nil {
+			logger.WithError(err).Warn("error while closing the SSLKEYLOGFILE")
```

Comment:
```suggestion
			logger.WithError(err).Warn("Error while closing the SSLKEYLOGFILE")
```
I think the convention is that log messages are capitalized, errors are not.